### DISCUSSION TITLE
[Backend] Golang + Fiber + GORM

### DISCUSSION THREAD

**Discussion by alpody** [realworld-apps/realworld#751] [Added implementations]
Conduit Backend API for Golang and Fiber + GORM  is done

repo: https://github.com/alpody/golang-fiber-realworld-example-app

**Comment by geromegrignon** [comment]
Hey @alpody,

Thanks for your implementation.
I opened a pull request in your repository.
Once merged, I'll approve it on CodebaseShow

**Reply by alpody** [reply]
I merged you pull request

**Reply by mvila** [reply]
Sorry @geromegrignon, I've approved @alpody's implementation before seeing this discussion. So it's done, and you don't have to worry about it.
